code/modules/ninja/suit/n_suit_verbs/ninja_stealth.dm

```diff
@@ -1 +1,43 @@
-// TODO
\ No newline at end of file
+
+/*
+
+Contents:
+- Stealth Verbs
+
+*/
+
+
+/obj/item/clothing/suit/space/space_ninja/proc/toggle_stealth()
```

Comment:
I doubt there's anyone left who remembers old ninjas....but.

One of the biggest strengths of old ninja was the fact they could go *fully* invisible, and were completely undetectable.